API/Controllers/PlayersController.cs

```diff
@@ -96,4 +96,22 @@ public async Task<IActionResult> ListAdminNotesAsync(int id)
 
         return Ok(await adminNoteService.ListAsync<PlayerAdminNote>(id));
     }
+
+    [HttpGet("{id:int}/friends")]
```

Comment:
Needs endpoint documentation, please see the other endpoints for examples.

Reply:
+1 Makes me wish I would stop putting off writing that docs page

Reply:
should've added a todo xd